integration_test/tests/wallet.rs

```diff
@@ -546,6 +585,23 @@ fn wallet__unload_wallet() {
     create_load_unload_wallet();
 }
 
+#[cfg(not(feature = "v20_and_below"))]
+#[test]
+fn wallet__send__modelled() {
+    use std::collections::BTreeMap;
+
+    let node = Node::with_wallet(Wallet::Default, &[]);
+    node.fund_wallet();
+    let address = node.client.new_address().expect("failed to create new address");
+
+    let mut outputs = BTreeMap::new();
+    outputs.insert(address.to_string(), 0.001);
+
+    let json: Send = node.client.send(&outputs).expect("send");
+    let model: Result<mtype::Send, SendError> = json.into_model();
+    model.unwrap();
```

Comment:
Cool. This works (I debug printed the result of the last unwrap but I'm baffled as to _why_ it works. Can you explain? The docs say 'data' is required but I don't understand why it works without it and what 'data' even is?

Reply:
`data` is not required, even though it doesn't say so in the docs. From what I read `It is for arbitrary hex-encoded data in a transaction, typically as an [OP_RETURN]  output e.g. { "data": "deadbeef" }`.

Reply:
ah, thanks.